skin/light/light.css

```diff
@@ -241,6 +241,10 @@
   border-left: 4px solid #0996f8 !important;
 }
 
+.tabbrowser-tab[pinned] + .tabbrowser-tab:not([pinned]) {
+  border-top: 1px solid black !important;
+}
```

Comment:
What happens when we drag the first unpinned tab down to a different spot in the tabstrip?